src/components/input/input.twig

```diff
@@ -13,6 +13,7 @@
     'size',
     'min',
     'max',
+    'rows',
```

Comment:
Does this need to forward data-min-rows as well?

Reply:
Answer: no

https://github.com/cloudfour/cloudfour.com-patterns/pull/716#discussion_r428293239